mysql-test/suite/rocksdb/t/partition.test

```diff
@@ -658,6 +658,21 @@ INSERT INTO trb1 VALUES
 
 ALTER TABLE trb1 ADD PRIMARY KEY (id);
 
+--echo #
+--echo # PS-5174: Check COMPRESSED COLUMNS support on partitioned tables
+--echo #
+CREATE COMPRESSION_DICTIONARY numbers ('');
```

Comment:
Hmm, we allow  empty dictionary?  we shouldn't have :(
Suggest to use some text.

Reply:
@satya-bodapati can you please file a bug re. empty dictionaries?

Reply:
@laurynas-biveinis done! https://jira.percona.com/browse/PS-5546